src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IUserNotificationServices.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    internal interface IUserNotificationServices
+    {
+        Task<bool> CheckPromptAsync(string promptMessage);
+
+        void NotifyFailureAsync(string renamedString);
+
+        Task<bool> CheckPromptForRenameAsync(string promptMessage);
```

Comment:
The methods that are associated with "rename" don't really belong here - as this is a general service unrelated to rename. However, given below - they are very simple methods that delegate on to other abstractions, is there any reason you need to mock them? ie, Why isn't CheckPromptForRenameAsync just a mock of CheckPromptAsync and IOptionsSettings?